CHANGELOG.md

```diff
@@ -19,6 +19,10 @@
   `snapshot-editor` tool to print MicrovmState of vmstate snapshot file in
   a readable format. Also made the `vcpu-states` subcommand available on
   x86_64.
+- [#4170](https://github.com/firecracker-microvm/firecracker/pull/4170):
+  Added support for vhost-user block devices. Firecracker implements
```

Comment:
It would be nice to include all related PRs here (at least known at this point).

Reply:
done